src/app.ts

```diff
@@ -22,7 +22,7 @@ prClient.getPullRequests().then(prs => {
         // Pull Request ID cell
         const tableCellId = document.createElement("td");
         tableCellId.setAttribute("sorttable_customkey", "" + pr.id);
-        tableCellId.innerHTML = "<a href='" + pr.url + "' target='_top'>Pull Request #" + pr.id + "</a>";
+        tableCellId.innerHTML = "<a href='" + (pr.baseUri + pr.projectName + "/_git/" + pr.repo + "/pullRequest/" + pr.id) + "' target='_top'>#" + pr.id + "</a>";
```

Comment:
This may need some url escaping as `projectName` and `repo` may contain spaces... maybe?

Reply:
This may apply to the other lines below as well.

Reply:
I wonder what other characters people have in their project and repo names... @nshenoy do you have something to test these corner cases (or normal cases :p ) out already?

Reply:
Given that this is exactly what the previous commit did, if you want to hand wave this away that is fair.

Reply:
@nshenoy If you check the changes to the service you'll see that `pr.url` is no longer exposed.  Before, it was the URL directly to the pull request.  Now it's the base URL of the Repository.  So in this case `pr.baseUrl` is now used twice (once to link to the PR like before, and once to link to the branches in question).